team_users_list.go

```diff
@@ -0,0 +1,94 @@
+package fivetran
+
+import (
+    "context"
+    "encoding/json"
+    "fmt"
+)
+
+// TeamUsersListService implements the Team Management, retrieve List all Teams.
+// Ref. https://fivetran.com/docs/rest-api/teams#listallteams
```

Comment:
This is definitely wrong comment

Reply:
This service refers to https://fivetran.com/docs/rest-api/teams#listallusermemberships